en/prod/Content/Topics/Apps/Apps_App_Integration_Wizard.htm

```diff
@@ -1,437 +1,10 @@
-                                            <p class="copyright">©  <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. All Rights Reserved. Various trademarks held by their respective owners.
-	</p>
-                                        </div>
-                                    </div>
-                                    <!-- Qualtrics setup to use with script below, replacing getfeedback -->
-                                    <div id="ZN_6fGBJ8J3VcIBymN">
-                                        <!--DO NOT REMOVE-CONTENTS PLACED HERE-->
-                                    </div>
-                                    <!-- ************************ Scripts ************************************* -->
-                                    <!-- Foundation -->
-                                    <script src="../../Resources/Scripts/js/app.js">
-                                    </script>
-                                    <script src="../../Resources/Scripts/js/vendor/what-input.js">
-                                    </script>
-                                    <script src="https://cdnjs.cloudflare.com/ajax/libs/foundation/6.4.4-rc1/js/foundation.min.js">
-                                    </script>
-                                    <script type="text/javascript" src="../../Resources/Scripts/js/vendor/qualtrics.js">
-                                    </script>
-                                    <!-- Coveo -->
-                                    <script src="https://static.cloud.coveo.com/searchui/v2.5395/js/CoveoJsSearch.Lazy.min.js" defer="defer">
-                                    </script>
-                                    <script src="../../Resources/Scripts/coveo-resources/js/cultures/en.js" id="coveoCultureScript" defer="defer">
-                                    </script>
-                                    <script src="../../Resources/Scripts/coveo-resources/js/Coveo.MadcapFlare.RequireJS.js" id="coveoRequireScript" defer="defer">
-                                    </script>
-                                    <script src="../../Resources/Scripts/coveo-resources/js/Coveo.MadcapFlare.Okta.js" id="coveoInitscript" defer="defer">
-                                    </script>
-                                </div>
-                            </div>
-                        </div>
-                    </div><a data-close="true"></a>
-                </div>
-            </div>
-        </div>
-    </body>
-</html>
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <meta name="robots" content="noindex, nofollow" />
+        <meta http-equiv="REFRESH" content="0.0;url=https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard.htm" />
```

Comment:
can you remind me why we don't have `/prod` anymore in our urls?

Reply:
`en/prod` is the old, legacy location. It was replaced with `en-us`, due to i18n/l10n considerations.